README.md

```diff
@@ -1,19 +1,41 @@
-# React render() Must Not Have Side Effects
+# React render() must not have side effects
 
-## Objectives
+???
 
-1. Review the requirements of `render()`
-2. Explain why `render()` must not have side effects
+# React `render()` Quiz
 
-## Overview
+?: `render()` is the only required method in a React component.
 
-This quiz is meant to give students a chance to review how to write a proper
-`render()` function. It's a review of pure functions and also a refresher about
-`render()` before we dive into props and state.
+[X] True.
+[ ] False.
 
-We can use this space to go over anything we might have missed in earlier units,
-and to really drive home how important `render()` is.
+?: `render()` can only return elements, not other React components.
 
-## Resources
+[ ] True.
+[X] False.
+
+?: `render()` must be a pure function.
+
+[X] True.
+[ ] False.
+
+?: `render()` can return multiple child elements.
+
+[ ] True.
+[X] False.
+
+?: It's okay if `render()` sometimes returns a different result when given the same input.
 
+[ ] True.
+[X] False.
+
+?: Which of the following expressions is _not_ a side effect?
+
+[ ] `console.log('Hello!');`
```

Comment:
I wonder if this is a good option to include. Technically, it's a side-effect, but developers certainly do use console.log in the render fn during development -- at least I do. Thoughts?